INSTALL.rst

```diff
@@ -4,6 +4,18 @@ INSTALLING DPLASMA
 
 .. contents:: Table of Contents
 
+
+TL;DR
+=====
+
+.. code:: bash
+
+  mkdir builddir && cd builddir
+  ${srcdir}/configure --with-hwloc --with-mpi --with-blas=Intel10_64lp_seq --disable-debug --prefix=$PWD/install
```

Comment:
Should we mention that this expects MKL to be available? We had at least one person run into this recently...

Reply:
- [ ] update tl;dr to link to the non simplified text when using other BLAS